WordPress/Classes/Utility/Universal Links/UniversalLinkRouter.swift

```diff
@@ -84,11 +84,21 @@ struct UniversalLinkRouter {
     /// Attempts to find a Route that matches the url's path, and perform its
     /// associated action.
     ///
-    func handle(url: URL) {
+    /// - parameter url: The URL to match against a route.
+    /// - parameter track: If false, don't post an analytics event for this URL.
+    ///
+    /// - returns: True if the route was handled, or false if it didn't match any routes.
+    ///
+    func handle(url: URL, shouldTrack track: Bool = true) {
         let matches = matcher.routesMatching(url)
 
         for matchedRoute in matches {
             matchedRoute.action.perform(matchedRoute.values)
         }
+
+        if track && matches.count > 0 {
```

Comment:
Should we track when we get a route we can't handle as well?

Reply:
Perhaps that'd make sense, yes – we would expect all URLs that launch the app to trigger a route, so it'd be good to know if we get one that doesn't. We could use the `failed` stat I just added?